MAINTAINERS.md

```diff
@@ -15,21 +15,11 @@ maintainers:
 * Gabriel Samfira @gabriel-samfira
 * Thilo Fromm @t-lo
 
-### coreos-overlay
```

Comment:
Please add me as a maintainer to scripts repo - this is where coreos-overlay and portage-stable got merged into.